src/api/app/views/webui/project/_buildstatus.html.erb

```diff
@@ -31,6 +31,11 @@
                                                           'arch_' + arch => 1, :project => params[:project], code => 1, :defaults => 0}, :rel => 'nofollow', :class => 'nowrap') %>
                     <% end %>
                   </td>
+                  <td>
+                    <% counts.each do |code, count| %>
+                        <img title="<%= get_package_status_description(code.to_s) %>" class="icons-help" alt="<%= get_package_status_description(code.to_s) %>" src='/assets/s.gif' /><br />
```

Comment:
If you change the `get_package_status_description` method as I suggested, you can get rid of the **.to_s** calls here